src/oasis-indexer/api.ts

```diff
@@ -22,8 +23,12 @@ function arrayify<T>(arrayOrItem: null | undefined | T | T[]): T[] {
   return arrayOrItem
 }
 
-export const useGetEmeraldTransactions: typeof generated.useGetEmeraldTransactions = (params?, options?) => {
-  const result = generated.useGetEmeraldTransactions(params, {
+export const useGetRuntimeTransactions: typeof generated.useGetRuntimeTransactions = (
+  runtime = Runtime.emerald,
```

Comment:
I would rather not have such defaults. It only increases chances of hidden sapphire bugs